config/routes.rb

```diff
@@ -5,9 +5,11 @@
         registrations: 'api/v1/registrations',
         sessions: 'api/v1/sessions'
       }
+      resources :users do
+        resources :reservations, only: %i[index create destroy]
+      end
       resources :doctors, only: [:index, :show, :new, :create]
       resources :reservations, only: [:index, :create, :destroy]
```

Comment:
Well done implementing the routes for reservations 👏 .  I noticed the routes for reservations is declared twice, but as reservations is associated with users, I will suggest removing line 12 as it is not needed.

Reply:
Thanks brother @oguntayomicheal .Resolved